Makefile

```diff
@@ -1,18 +1,21 @@
 CC=clang
 CXX=clang++
 LLVM_CONFIG=llvm-config
+CFORMAT ?= clang-format
+CPPFILES = $(wildcard *.cpp)
+CFILES = $(wildcard tests/*.c)
 export
 
 all: manual-split find-and-split-static split-llvm-extract
 
 manual-split: manual-split.cpp
-	$(CXX) -fuse-ld=lld -glldb $(shell $(LLVM_CONFIG) --link-shared --cppflags --ldflags --system-libs --libs) -std=c++17 -fno-exceptions $< -o ./manual-split
+	$(CXX) -glldb $(shell $(LLVM_CONFIG) --link-shared --cppflags --ldflags --system-libs --libs) -std=c++17 -fno-exceptions $< -o ./manual-split
```

Comment:
I think we should break the `Makefile` out into either 1) its own commit or 2) merge it with a previous commit as I assume you can't build the previous commits without this updated command-line invocation?